src/test/scala/com/mozilla/telemetry/streaming/TestErrorAggregator.scala

```diff
@@ -57,7 +58,8 @@ class TestErrorAggregator extends FlatSpec with Matchers with BeforeAndAfterAll
   "The aggregator" should "sum metrics over a set of dimensions" in {
     import spark.implicits._
     val messages =
-      (TestUtils.generateCrashMessages(k)
+      (TestUtils.generateCrashMessages(k - 1)
+        ++ TestUtils.generateCrashMessages(1, customPayload = Some(""""metadata":{"StartupCrash": "1"}"""))
```

Comment:
Once https://github.com/mozilla-services/mozilla-pipeline-schemas/pull/146 is merged, `metadata` should be added by default to the test crash pings. Good to go for now though

Reply:
Done.